### ISSUE TITLE
az webapp config ssl (show, list) doesn't return thumbprint

### ISSUE THREAD

**Issue by fifron** [Azure/azure-cli#19581]
Today I noticed that commands:
`az webapp config ssl show ...`
`az webapp config ssl list ...`
don't return thumbprint value. Thumbprint is empty:
"thumbprint": ""

I need thumbprint value for generating SSL + certificate for my custom domain.

My sequence is:
az webapp config ssl create ...
az webapp config ssl show ... (read thumbprint. Sometimes you have to wait few minutes)
az webapp config ssl bind ... (use thumbprint)

Has the way how to read thumbprint changed?

## Expected Behavior
I know "This command is in preview and under development".
But I expect I am able to generate SSL

I am able to bind SSL via Azure web portal. Here it works.

## Environment Summary
```
macOS-11.5.2-x86_64-i386-64bit
Python 3.8.7
Installer: HOMEBREW

azure-cli 2.19.1

Extensions:
resource-graph 1.1.0
```

**Comment by yonzhan** [comment]
route to service team

**Comment by panchagnula** [comment]
@fifron I assuming your question is specific to ManagedCerts, which doesn't have Thumbprint - for Managed Certs Thumbprint is not required. The managed certs command update will be released next released for this. In the meantime you can use the following , sorry for the inconvenience. The API changes deployment & the CLI deployment didn't coincide as planned.
Using CLI
az rest --method PUT --uri 'https://management.azure.com/subscriptions/{subId}/resourceGroups/{rgName}/providers/Microsoft.DomainRegistration/domains/{domainName}/transferOut?api-version=2021-02-01'

Using PowerShell
Invoke-AzRestMethod -Path "/subscriptions/{subId}/resourcegroups/{rename}/providers/Microsoft.DomainRegistration/domains/{domainName}/transferout?api-version=2021-02-01" -Method PUT

**Comment by fifron** [comment]
> @fifron I assuming your question is specific to ManagedCerts, which doesn't have Thumbprint - for Managed Certs Thumbprint is not required. The managed certs command update will be released next released for this. In the meantime you can use the following , sorry for the inconvenience. The API changes deployment & the CLI deployment didn't coincide as planned.
> Using CLI
> az rest --method PUT --uri '[https://management.azure.com/subscriptions/{subId}/resourceGroups/{rgName}/providers/Microsoft.DomainRegistration/domains/{domainName}/transferOut?api-version=2021-02-01](https://management.azure.com/subscriptions/%7BsubId%7D/resourceGroups/%7BrgName%7D/providers/Microsoft.DomainRegistration/domains/%7BdomainName%7D/transferOut?api-version=2021-02-01)'
> 
> Using PowerShell
> Invoke-AzRestMethod -Path "/subscriptions/{subId}/resourcegroups/{rename}/providers/Microsoft.DomainRegistration/domains/{domainName}/transferout?api-version=2021-02-01" -Method PUT

@panchagnula You are right, I use ManagedCerts. I am sure Thumbprints were working in the past. This change must be from last days.
Do you know the date of new release? Will be new command placed under "az webapp config"?

I have tried your command DomainRegistration, but unsuccessfully.
I expect that it is because I use common AppService creating with:
az webapp create ...
az webapp config hostname add ...
so I don't have any domain.

**Comment by Shrinathbcone** [comment]
**@panchagnula** Thumbprint and SSL binding was working fine few days back and we have been used as an input parameter in our automation deployment code.
 
Our deployment cycle is wedged as this is impacting customer service and customer engagement.
Kindly let us know if there is an alternative approach to get thumbprint since we are using ManagedCerts for App services.

**Comment by panchagnula** [comment]
> **@panchagnula** Thumbprint and SSL binding was working fine few days back and we have been used as an input parameter in our automation deployment code.
> 
> Our deployment cycle is wedged as this is impacting customer service and customer engagement.
> Kindly let us know if there is an alternative approach to get thumbprint since we are using ManagedCerts for App services.

@Shrinathbcone see the CLI command offered above to use for binding without needing Thumbprint as alternate approach
pasting it here again az rest --method PUT --uri 'https://management.azure.com/subscriptions/{subId}/resourceGroups/{rgName}/providers/Microsoft.DomainRegistration/domains/{domainName}/transferOut?api-version=2021-02-01'
